timon/configure.py

```diff
@@ -258,14 +329,17 @@ def apply_config(options):
 
     statefile = os.path.join(workdir, cfg.get('statefile', 'timon_state.json'))
     cfg['statefile'] = options.statefile or statefile
-
     if do_check:
         print("CHECK_CFG not implemented so far")
         return
-
+    if "webif" in cfg:
```

Comment:
I suggest, that the json file should always contain a webif entry (just an empty dict for the moment),

but this might allow to have one less check in the javascript or in potential web server code.